src/content/quickstart/kind/_index.md

```diff
@@ -1,30 +1,41 @@
 ---
 date: 2020-03-17T13:36:18+01:00
-title: "KIND"
+title: "KIND (Local Environment)"
 weight: 100
 ---
 
-## KIND
+## Locally Testing With KIND
```

Comment:
@Jaanki was planning to work on this section, the idea was about referencing how to deploy just the "clusters", and then, from there let the user handle it with subctl, so they learn more about the process.

Reply:
Can you coordinate with @Jaanki ?

Reply:
I was thinking of adding subctl steps after "make deploy". @mkolesnik you can add those as part of this PR or I can push a separate one ater this is merged. WDYT @mangelajo

Reply:
You'll notice I'm not changing the content too much. @Jaanki can still work on this section and improve it, these are more technical changes..

Reply:
> I was thinking of adding subctl steps after "make deploy". @mkolesnik you can add those as part of this PR or I can push a separate one ater this is merged. WDYT @mangelajo

Specifically, after `make deploy` you'll already have a deployed submariner.. If you want to detail how to do it more manually you can change this to `make clusters` and add the subctl calls, but since this is *quickstart* I think we should be concise.
Either way, deploying with subctl is already documented under the deployment section, so not sure it should be replicated here as well..

Reply:
> I was thinking of adding subctl steps after "make deploy". @mkolesnik you can add those as part of this PR or I can push a separate one ater this is merged. WDYT @mangelajo

That would work, but you need to change deploy to 'clusters' :-) otherwise the job of the user will be done already :)